### DISCUSSION TITLE
Initial Setup: Analytics for Plausible.io Analytics Dashboard

### DISCUSSION THREAD

**Discussion by ACPK** [plausible/analytics#582] [Anything Else]
What are your thoughts on having the initial app that's created for the self-hosted version be the analytics for how users are using the Plausible Analytics app? This could also be helpful in seeing how Plausible events and goals are used.

For example, the analytics events (i.e., goals) would be register, add website, schedule weekly email, etc.

**Comment by KristerV** [comment]
But you would only see your own path at first. Then I'd want to get rid of it and it's a headache finding the button.

I would like this idea if the concept of this tool was complicated, but it's not. Everyone knows Google Analytics.
